cmake/multi_image.cmake

```diff
@@ -125,6 +125,22 @@ function(add_child_image_from_source)
   set(${ACI_NAME}_BOARD ${ACI_BOARD})
 
   message("\n=== child image ${ACI_NAME} - ${ACI_DOMAIN}${inherited} begin ===")
+
+  # Look for a folder named 'child_image_overlay' and apply all relevant
+  # overlay files. An overlay file will be applied if it is located in the
+  # root of the child_image_overlay folder and matches ${ACI_NAME}-overlay.conf
+  # or if it is stored in
+  # 'child_image_overlay/parent_board_name/${ACI_NAME}-overlay.conf'
+  set(child_conf ${CMAKE_SOURCE_DIR}/child_image_overlay)
```

Comment:
Today, Zephyr natively supports:
```
<sample>
  |--- prj.conf
  |--- prj_<desc>.conf
  |--- boards
         |--- <board>.conf
         |--- <board>_<desc>.conf
```
and I think we should have similar support for each child-image. (See other message)

So maybe:
```suggestion
  set(child_conf ${CMAKE_SOURCE_DIR}/child_image) 
```
or
```suggestion
  set(child_conf ${CMAKE_SOURCE_DIR}/child_image_configurations) 
```
would be better.